views/login/sin.pug

```diff
@@ -8,6 +9,9 @@ block content
 
   h1 #{title}
 
+  +banner('check', '')
```

Comment:
Can you pass `null` in instead? Otherwise it looks like the empty space is doing something.